src/components/Footer.jsx

```diff
@@ -58,17 +57,18 @@ class SiteFooter extends React.Component {
           >
             <img
               style={{ maxWidth: 150 }}
-              src={logo || FooterLogo}
+              src={logo || process.env.LOGO_TRADEMARK_URL}
               alt={intl.formatMessage(messages['footer.logo.altText'])}
             />
           </a>
           <div className="flex-grow-1" />
-          {showLanguageSelector &&
-            <LanguageSelector
-              options={supportedLanguages}
-              onSubmit={onLanguageSelected}
-            />
-          }
+          {showLanguageSelector
```

Comment:
nit: 

```
{showLanguageSelector && (
  <LanguageSelector ... />
)}
```

Reply:
husky linting is the only reason I changed this to the way it is 🐕